src/me/john000708/barrels/listeners/DisplayListener.java

```diff
@@ -1,35 +1,45 @@
 package me.john000708.barrels.listeners;
 
+import me.john000708.barrels.Barrels;
+import me.john000708.barrels.DisplayItem;
+
 import org.bukkit.Bukkit;
 import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
 import org.bukkit.event.Listener;
-import org.bukkit.event.entity.EntityPickupItemEvent;
+import org.bukkit.event.block.BlockPlaceEvent;
 import org.bukkit.event.inventory.InventoryPickupItemEvent;
-
-import me.john000708.barrels.Barrels;
+import org.bukkit.event.player.PlayerPickupItemEvent;
 
 /**
  * Created by John on 10.05.2016.
  */
+@SuppressWarnings("deprecation")
 public class DisplayListener implements Listener {
 	
     public DisplayListener() {
         Bukkit.getPluginManager().registerEvents(this, Barrels.plugin);
     }
 
     @EventHandler
-    public void onPickpup(EntityPickupItemEvent e) {
-        if (!e.getItem().hasMetadata("no_pickup") && e.getItem().getItemStack().hasItemMeta() && e.getItem().getItemStack().getItemMeta().hasDisplayName() && e.getItem().getItemStack().getItemMeta().getDisplayName().startsWith("�6�lB4R3L - �eITEM")) {
+    public void onPickpup(PlayerPickupItemEvent e) {
```

Comment:
Changing this back to a deprecated event?
By doing this you remove the support for other mobs e.g. zombies accidentally picking up the display item...